examples/complete/main.tf

```diff
@@ -463,59 +412,18 @@ resource "aws_s3_object" "hr_data" {
   tags = local.tags
 }
 
-resource "aws_iam_role" "s3_role" {
-  name        = "${local.name}-s3"
-  description = "Role used to migrate data from S3 via DMS"
-
-  assume_role_policy = jsonencode({
-    Version = "2012-10-17"
-    Statement = [
-      {
-        Sid    = "DMSAssume"
-        Action = "sts:AssumeRole"
-        Effect = "Allow"
-        Principal = {
-          Service = "dms.${data.aws_partition.current.dns_suffix}"
-        }
-      },
-    ]
-  })
-
-  inline_policy {
-    name = "${local.name}-s3"
-
-    policy = jsonencode({
-      Version = "2012-10-17"
-      Statement = [
-        {
-          Sid      = "DMSRead"
-          Action   = ["s3:GetObject"]
-          Effect   = "Allow"
-          Resource = "${module.s3_bucket.s3_bucket_arn}/*"
-        },
-        {
-          Sid      = "DMSList"
-          Action   = ["s3:ListBucket"]
-          Effect   = "Allow"
-          Resource = module.s3_bucket.s3_bucket_arn
-        },
-      ]
-    })
-  }
-
-  tags = local.tags
-}
-
 module "msk_cluster" {
   source  = "clowdhaus/msk-kafka-cluster/aws"
```

Comment:
```suggestion
  source  = "terraform-aws-modules/msk-kafka-cluster/aws"
```

Reply:
updated in https://github.com/terraform-aws-modules/terraform-aws-dms/pull/42/commits/6ef37ea00686d51687526f60f6bb9765bea7ef18